geometry_msgs/PoseStamped goal
---
bool ok
geometry_msgs/PoseStamped[] path
